data/storage_proof.in

```diff
@@ -0,0 +1,9 @@
+{
+    "provider_url": "RPC_URL",
```

Comment:
you can use `std::env::var` to get RPC_URL from environmental variable, to avoid potential leakage